pyicloud/services/calendar.py

```diff
@@ -41,7 +41,12 @@ def refresh_client(self, from_dt=None, to_dt=None):
         today = datetime.today()
         first_day, last_day = monthrange(today.year, today.month)
         if not from_dt:
-            from_dt = datetime(today.year, today.month, first_day)
+            # fix for issue #132
```

Comment:
For future contributions, note that the content of this comment should typically be part of the commit message, not the code itself.